tools/fastqc/tool_dependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<tool_dependency>
+  <package name="FastQC" version="0.11.2">
+      <repository changeset_revision="4b65f6e39cb0" name="package_fastqc_0_11_2" owner="devteam" toolshed="https://toolshed.g2.bx.psu.edu" />
+    </package>
+</tool_dependency>
```

Comment:
Since the tool will run remotely, do you recommend installing the tool on the central site and then syncing the Galaxy directory across all sites? Or installing the tool separately, but point to it using environment variables?